src/ensembl/src/content/app/browser/browser-bar/BrowserBar.test.tsx

```diff
@@ -56,21 +53,21 @@ describe('<BrowserBar />', () => {
       const renderedBrowserBar = mount(
         renderBrowserBar({ ensObject: createEnsObject('gene') })
       );
-      expect(renderedBrowserBar.find(GeneSummaryStrip).length).toBe(1);
+      expect(renderedBrowserBar.find(FeatureSummaryStrip).length).toBe(1);
     });
 
     test('renders RegionSummaryStrip if focus object is region', () => {
       const renderedBrowserBar = mount(
         renderBrowserBar({ ensObject: createEnsObject('region') })
       );
-      expect(renderedBrowserBar.find(RegionSummaryStrip).length).toBe(1);
+      expect(renderedBrowserBar.find(FeatureSummaryStrip).length).toBe(1);
```

Comment:
As mentioned in the comment above, this test no longer belongs to BrowserBar tests (but does belong to FeatureSummaryStrip tests).